bin/cli.js

```diff
@@ -2,12 +2,31 @@
 
 'use strict';
 
+const yargs = require('yargs');
 const readline = require('readline');
 
 const Cli = require('../lib/cli/cli');
 
 let rl = null;
 
+const args = yargs
+  .option('verbose', {
+    alias: 'v',
+    count: true,
```

Comment:
Why do you use `count` option here and not `type` like in other options definitions?

Reply:
I need to use `count` so that when you pass `-vv`, `verbose` will be equal to 2. I do not use `type` here because this option does not have any value associated with but instead, we simply count the number of occurrences of this option.

Reply:
I meant that you could've provided `type: 'count'` option instead of using `count` option, to make it consistent with other options definitions.